.github/workflows/performance.yml

```diff
@@ -37,6 +37,13 @@ jobs:
             - name: Setup Node.js and install dependencies
               uses: ./.github/setup-node
 
+            - name: Install NVM
+              run: |
+                  curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+                  export NVM_DIR="$HOME/.nvm"
+                  [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+                  nvm -v
+
```

Comment:
```suggestion
            - name: Setup Node.js and install dependencies
              uses: ./.github/setup-node
              with:
                node-version-file: '.nvmrc'
```

I don't think installing _nvm_ is required, I bumped into [this](https://github.com/actions/setup-node#usage) yesterday:
```
# File containing the version Spec of the version to use.  Examples: .nvmrc, .node-version, .tool-versions.
    # If node-version and node-version-file are both provided the action will use version from node-version. 
    node-version-file: ''
```

The above would only work for the version in the `.nvmrc` file, so would rule out using it in an CI matrix configuration trying to test with multiple Node.js versions 

https://github.com/WordPress/gutenberg/blob/4e9b121b269c315a0c159411bcd03a8a55dc0590/.nvmrc#L1

Reply:
It is required for the performance tests because performance tests compare different branches using a custom script and we need to change the node version ourselves within that script. If we want to use the Github action to switch versions within the job, we'll have to completely rewrite the performance.js command and move it to the GitHub action. I don't think it's worth the hassle, that script is way too complex for that.

Reply:
We probably need to do something similar with the "Compressed Size" workflow (in `bundle-size.yml`) because it currently fails with warnings about old lockfile:
```
Installing using npm ci
/opt/hostedtoolcache/node/18.16.1/x64/bin/npm cinpm WARN old lockfile 
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
```
Apparently the job is installing packages in `trunk` with Node 18, the new version.